recipe/recipe.yaml

```diff
@@ -12,19 +12,30 @@ source:
   - if: polars_variant == 'polars'
     then:
       url: https://pypi.org/packages/source/p/polars/polars-${{ version }}.tar.gz
-      sha256: 94fcb0216b56cd0594aa777db1760a41ad0dfffed90d2ca446cf9294d2e97f02
+      sha256: d2acb71fce1ff0ea76db5f648abd91a7a6c460fafabce9a2e8175184efa00d02
+      # https://github.com/prefix-dev/rattler-build/issues/1000
+      # patches:
+      #   - if: target_platform == 'linux-64' or target_platform == 'win-64'
+      #     then: cargo-use-argminmax-fork.patch
```

Comment:
Remove?

Reply:
this was intentional. will uncomment it once the issue has been resolved. currently, i'm manually applying this patch using `sed`

Reply:
Yep, understood that. Didn't get that this is a workaround for the rattler-build issue.